src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -3632,4 +3632,10 @@ In certain complex graphs or host program scenarios, Automatic mode may cause in
   <data name="CantExportWorkspaceAsImageNotValidMessage" xml:space="preserve">
     <value>The workspace cannot be exported as an image because it contains nodes that are too far away from each other.</value>
   </data>
-</root>
+  <data name="PreferencesViewSettingIncludeTimestampExportPath" xml:space="preserve">
+    <value>Include TImestamp in Export Path</value>
```

Comment:
can you change `TImestamp` to `Timestamp`

Reply:
Done